x/challenge/abci.go

```diff
@@ -89,7 +91,9 @@ func EndBlocker(ctx sdk.Context, keeper k.Keeper) {
 			continue
 		}
 		if sp.Status != sptypes.STATUS_IN_SERVICE && sp.Status != sptypes.STATUS_GRACEFUL_EXITING {
-			continue
+			if !ctx.IsUpgraded(upgradetypes.Hulunbeier) || sp.Status != sptypes.STATUS_FORCED_EXITING {
```

Comment:
`STATUS_FORCED_EXITING` is comes together `Hulunbeier` fork
maybe, we can just check the status.

Reply:
fixed